plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -893,6 +901,10 @@ public Optional<List<SchemaTableName>> getAllTables()
                         }
                     }));
         }
+        catch (TTransportException e) {
```

Comment:
Why this change? Can be transient exception?

Reply:
This is regardless @Praveen2112 comment https://github.com/trinodb/trino/pull/18274#issuecomment-1653352216
I would like to fallback to previous mechanism (so I return `Optional.empty` like this method is not implemented) in case of error from https://github.com/trinodb/trino/issues/18111 Maybe there is better solution for that.

Now when I'm thinking I would extract this (and the test for it) as separate commit for better visibility. I will do that

Reply:
So this is a separate change.

Reply:
Extracted to separate commit